vulcanai/datasets/tabular_data_utils.py

```diff
@@ -86,6 +91,22 @@ def create_label_encoding(df, column_name, ordered_values):
     return df
 
 
+def create_one_hot_encoding(df, column_name, prefix_sep="@"):
+    """
+    Create one-hot encoding for the given column.
+    Parameters:
+        column_name: String
+            The name of the column you want to one-hot encode
+        prefix_sep: String default("@")
+            The prefix used when creating a one-hot encoding
```

Comment:
it may be redundant, but can we add a small "Returns" section in the docs to indicate that the returned df will have a new column that will represent that one hot encoding? or does it stay the same? it would change how the df should be used after passing it through here